en/config/actuators.md

```diff
@@ -0,0 +1,116 @@
+- Tiltable motors are then assigned to one of the tilt servos.
+
+![Tilt Axis](../../assets/config/tilt_axis.png)
+
+### Reversible Motors
+If reversible motor(s) are used (i.e. motor spins in direction 1 for lower output range and in direction 2 for the upper half), make sure to select the **Reversible** checkbox for those motor(s).
+Also ensure the ESC is configured appropriately (e.g. 3D mode enabled for DShot ESCs, which can be achieved via [DShot commands](../peripherals/dshot.md#commands)).
+
+
+## Actuator Outputs
+The actuators and any other output function can be assigned to any of the physical outputs.
+Each output has its own tab, e.g. the PWM MAIN or AUX output pins.
+
+PWM outputs are grouped according to the hardware groups of the autopilot.
+Each group allows to configure the PWM rate or DShot/Oneshot (if supported).
+
+:::note
+For boards with MAIN and AUX, prefer the AUX pins over the MAIN pins for motors, as they have lower latency.
+:::
+
+The AUX pins have additional configuration options for camera capture/triggering.
+Selecting these requires a reboot before they are applied.
+
+
+## Actuator Testing
+When testing actuators, make sure that:
+- Motors spin at the minimum. The sliders snap into place at the lower end: at the bottom, the motor is turned off (disarmed), whereas the next slider position commands the minimum thrust.
+  Adjust the minimum output value such that the motors spin at that slider position.
+  :::note
+  VTOL's will automatically turn off motors pointing upwards during forward flight.
+  :::
+- Servos move into the direction of the convention described above.
+  :::note
+  A trim value can be configured for control surfaces, which is also applied to the test slider.
+  :::
+
+Note the following behaviour:
+- If a safety button is used, it must be pressed before actuator testing is allowed.
+- The kill-switch can still be used to stop motors immediately.
+- Servos do not actually move until the corresponding slider is changed.
+- The parameter [COM_MOT_TEST_EN](../advanced_config/parameter_reference.md#COM_MOT_TEST_EN) can be used to completely disable actuator testing.
+- On the shell, [actuator_test](../modules/modules_command.md#actuator-test) can be used as well.
+
+### Reversing Motors
+:::note
+The *spin direction* checkbox for the geometry does not actually change the motor spin direction. The setting must match with the physical spin direction of the motor.
+:::
+
+If one or more of the motors do not turn in the correct direction according to the configured geometry, they must be reversed.
+There are several options:
+- If the ESCs are configured as [DShot](../peripherals/dshot.md) you can reverse the direction via UI (**Set Spin Direction** buttons). Note that the current direction cannot be querried, so you might have to try both options.
```

Comment:
Typo. Again  we need to show somewhere setting spin directions.

```suggestion
- If the ESCs are configured as [DShot](../peripherals/dshot.md) you can reverse the direction via UI (**Set Spin Direction** buttons). Note that the current direction cannot be queried, so you might have to try both options.
```

Reply:
Applied